README.md

```diff
@@ -15,7 +15,7 @@ conformance('ISC OR GPL-2.0-with-GCC-exception')
 
 ## What
 
-This module will spit out an object at you with a suite of information about a SPDX license expression you pass in. In general, it will look something like this:
+This module will spit out an object at you with a suite of information about an SPDX license expression you pass in. In general, it will look something like this:
```

Comment:
I believe SPDX is pronounced as `ES-PEE-DEE-EX`, due to the `ES`, it should be `an` instead of `a` I think